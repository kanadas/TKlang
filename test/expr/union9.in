((\a b -> if a + b == 0 then 1 @ 'a' else 2 @ a + b) :: int -> int -> char + int) 1 2

