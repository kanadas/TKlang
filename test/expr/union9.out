expr/union9.in
2@3
